site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+```
+
+Refresh your browser and you should now see two stumbling zombies moving towards one another. Feel free to have a play with the params for rotation / timings to get it just how you like it.
+
+![Zombie-Walk](./walking.gif)
+
+Notice that these two new animations have `iterations` set to `Infinity`. This ensures that they will play indefinitely, or until the first animation completes and `this._play` is set to false.
+
+{% section %}
+
+## Joining Animations
+
+Lets finish off by adding some hearts that appear once the Zombies have reached the middle of the screen. We want them to float upto the top of the screen whilst changing size. We can do this by returning an array of `AnimationProperties` rather than a single object.
+
+{% instruction 'Add hearts to Zombies.ts' %}
+
+``` typescript
+private _playHearts = false;
+private _numHearts = 5;
+
+private _onAnimationFinish() {
+	this._play = false;
+
+	// add toggle for `_playHearts`
+	this._playHearts = true;
+
+	this.invalidate();
+}
+
+private _onHeartsFinish() {
+	if (this._playHearts = true) {
+		this._playHearts = false;
+		this.invalidate();
+	}
+}
+
+private _getHeartAnimation(id: string, sequence: number, play: boolean): AnimationProperties[] {
+	const delay = sequence * 500;
+	const leftOffset = Math.floor(Math.random() * 400) - 200;
+
+	return [
+		{
+				id: `${id}FloatAway`,
+				effects: [
+					{ opacity: 0, marginTop: '0', marginLeft: '0px' },
+					{ opacity: 0.8, marginTop: '-300px', marginLeft: `${1- leftOffset}px` },
+					{ opacity: 0, marginTop: '-600px', marginLeft: `${leftOffset}px` }
+				],
+				timing: {
+					duration: 1500,
+					delay,
+				},
+				controls: {
+					play: this._playHearts,
+					onFinish: sequence === this._numHearts -1 ? this._onHeartsFinish : undefined
+				}
+		},
+		{
+			id: `${id}Scale`,
+			effects: [
+				{ transform: 'scale(1)' },
+				{ transform: 'scale(0.8)' },
+				{ transform: 'scale(1)' },
+				{ transform: 'scale(1.2)' },
+				{ transform: 'scale(1)' }
+			],
+			timing: {
+				duration: 500,
+				iterations: Infinity,
+				delay,
+				easing: 'ease-in-out'
+			},
+			controls: {
+				play: this._playHearts
+			}
+		}
+	];
+}
+
+private _getHearts(): HNode[] {
+	const hearts = [];
+	let play = false;
+	let i;
+	for (i = 0; i < this._numHearts; i++) {
+		const key = `heart${i}`;
+		hearts.push(v('div', { classes: css.heart, key }));
+		this.meta(WebAnimation).animate(key, this._getHeartAnimation(key, i, play));
+	}
+	return hearts;
+}
+
+// add this as the last child of the `root` HNode in render
+v('div', { classes: css.heartsHolder }, this._getHearts())
+
+```
+
+Now you should see the hearts appearing and floating away when the zombies get to the middle of the screen.
+
+{% section %}
+
+## Controlling our animations
+
+Due to the reactive nature of the Dojo 2 `WebAnimation` meta, we can control our animation by changing the properties passed. For example we can change the `playbackRate` programatically.
```

Comment:
"by changing the properties passed" -> by changing the properties passed into ...